### PR TITLE
zh-translation:/docs/ops/common-problems/observability-issues/index.md

### PR DESCRIPTION
**Pull request by JasonRD**
ref: https://github.com/servicemesher/istio-official-translation/issues/1238

### CODE REVIEW

**Comment by wangzewang** [comment]
当前翻译的文件在英文版中已经被更新，请翻译最新版本的文件[https://github.com/istio/istio.io/blob/master/content/en/docs/ops/common-problems/observability-issues/index.md](https://github.com/istio/istio.io/blob/master/content/en/docs/ops/common-problems/observability-issues/index.md)

**Comment by JasonRD** [comment]
> 当前翻译的文件在英文版中已经被更新，请翻译最新版本的文件https://github.com/istio/istio.io/blob/master/content/en/docs/ops/common-problems/observability-issues/index.md

**Comment by JasonRD** [comment]
Changed content  to latest.

File: content/zh/docs/ops/common-problems/observability-issues/index.md [hunk 1]
```diff
@@ -1,55 +1,66 @@
 ---
-title: Observability Problems 
-description: Dealing with telemetry collection issues.
+title: 可观察性问题
```
Comment (wangzewang) [inline content/zh/docs/ops/common-problems/observability-issues/index.md]: Observability翻译为可观测性

File: content/zh/docs/ops/common-problems/observability-issues/index.md [hunk 2]
```diff
@@ -1,55 +1,66 @@
...
+如果 Istio Dashboard 或 Prometheus 查询未显示预期的指标，则上述流程的任何步骤都可能会出现问题。以下部分提供了对每个步骤进行故障排除的说明。
 
-### Verify Mixer is receiving Report calls
+### （如果需要）验证 Istio CNI pod 正在运行{#verify-Istio-CNI-pods-are-running}
 
-Mixer generates metrics to monitor its own behavior. The first step is to check these metrics:
+在 Kubernetes Pod 生命周期设置网络期间，Istio CNI 插件会对 Istio 网格 Pod 执行流量重定向，从而用户在 Istio  网格中部署 Pod 时不需要 [`NET_ADMIN`能力需求](/zh/docs/ops/setup/required-pod-capabilities/)。 Istio CNI 插件主要用来替代 `istio-init` 容器的一些功能。
 
-1. Establish a connection to the Mixer self-monitoring endpoint for the `istio-telemetry` deployment. In Kubernetes environments, execute the following command:
+1. 验证 `istio-cni-node` pod 正在运行：
```
Comment (wangzewang) [inline content/zh/docs/ops/common-problems/observability-issues/index.md]: pod=>pods

File: content/zh/docs/ops/common-problems/observability-issues/index.md [hunk 3]
```diff
@@ -1,55 +1,66 @@
...
+    $ kubectl -n kube-system get pod -l k8s-app=istio-cni-node
+    {{< /text >}}
+
+1. 如果 `PodSecurityPolicy` 在您的集群上已经启用，请确保 `istio-cni` 服务账号可以使用具有 [`NET_ADMIN`能力需求](/zh/docs/ops/setup/required-pod-capabilities/)的 `PodSecurityPolicy`。
+
+### 确认 Mixer 可以收到指标报告的调用{#verify-mixer-is-receiving-report-calls}
+
+Mixer 会生成指标来监控它自身行为。首先，检查这些指标：
+
+1. `istio-telemetry` Deployment 对外暴露 Mixer 自监控端点。在 Kubernetes 环境中，执行以下命令：
```
Comment (wangzewang) [inline content/zh/docs/ops/common-problems/observability-issues/index.md]: endpoint不翻译

File: content/zh/docs/ops/common-problems/observability-issues/index.md [hunk 4]
```diff
@@ -1,55 +1,66 @@
...
 
-1. Verify successful report calls. On the Mixer self-monitoring endpoint (`http://localhost:15014/metrics`), search for `grpc_io_server_completed_rpcs`. You should see something like:
+1. 验证上报成功调用。通过 Mixer 自监控端点 (`http://localhost:15014/metrics`) 查询 `grpc_io_server_completed_rpcs`，您应该能看到类似的东西：
 
     {{< text plain >}}
     grpc_io_server_completed_rpcs{grpc_server_method="istio.mixer.v1.Mixer/Report",grpc_server_status="OK"} 2532
     {{< /text >}}
 
-    If you do not see any data for `grpc_io_server_completed_rpcs` with a `grpc_server_method="istio.mixer.v1.Mixer/Report"`, then Envoy is not calling Mixer to report telemetry.
+    如果你没有发现带有 `grpc_server_method="istio.mixer.v1.Mixer/Report"` 的数据，说明 Envoy 没有调用 Mixer 上报遥测数据。
```
Comment (wangzewang) [inline content/zh/docs/ops/common-problems/observability-issues/index.md]: `grpc_io_server_completed_rpcs` 没有翻译

Comment (JasonRD) [reply content/zh/docs/ops/common-problems/observability-issues/index.md]: grpc_io_server_completed_rpcs不需要翻译

Comment (wangzewang) [reply content/zh/docs/ops/common-problems/observability-issues/index.md]: 译文中删掉了 `grpc_io_server_completed_rpcs `

Comment (JasonRD) [reply content/zh/docs/ops/common-problems/observability-issues/index.md]: 谢谢。漏掉了

**Pull request merged**